lib/screens/conversations_screen.dart

```diff
@@ -34,25 +75,37 @@ class ConversationsScreen extends StatelessWidget {
     }
   }
 
-  final bool addUser;
-  ConversationsScreen({this.addUser});
-
   @override
   Widget build(BuildContext context) {
+    userData = Provider.of<Users>(context);
     final Conversations conversationData = Provider.of<Conversations>(context);
     Map<int, Conversation> conversations = conversationData.conversations;
-    Users userData = Provider.of<Users>(context);
-    User currentUser = userData.getCurrentUser();
+    User currentUser = userData.currentUser;
```

Comment:
This occurs often.

Reply:
It's needed often

Reply:
Looking through the code I think it's needed less than its being used I'll shuffle things around